WinImageBuilder.psm1

```diff
@@ -540,11 +540,13 @@ function Add-PackageToImage {
         [Parameter(Mandatory=$true)]
         [string]$winImagePath,
         [Parameter(Mandatory=$true)]
-        [string]$packagePath
+        [string]$packagePath,
+        [Parameter(Mandatory=$false)]
+        [boolean]$ignoreErrors
     )
     Write-Log ('Adding packages from "{0}" to image "{1}"' -f $packagePath, $winImagePath)
     & Dism.exe /image:${winImagePath} /Add-Package /Packagepath:${packagePath}
-    if ($LASTEXITCODE) {
+    if ($LASTEXITCODE -and !$ignoreErrors) {

```

Comment:
The patch looks good in general.

can you add a log info to print that dism exited with errors for the package installed?
This addition can help you debug stuff in case of a dism failure and it makes sure the code is transparent to the user.